plugins/by-name/conform-nvim/default.nix

```
@@ -43,6 +48,26 @@ lib.nixvim.plugins.mkNeovimPlugin {
         "formatOnSave"
       ];
 
+  extraOptions = {
+    autoInstall = {
+      enable = lib.mkEnableOption "
+        automatic installation of formatters listed in `settings.formatters_by_ft` and `settings.formatters`
+      ";
+      overrides = lib.mkOption {
+        type = with types; attrsOf (nullOr package);
+        default = { };
+        example = {
+          "treefmt" = null;
+          "pyproject-fmt" = lib.literalExpression "pkgs.python312Packages.pyproject-parser";
+        };
```

You can't nest a literalExpression within a value. It needs to be assigned directly to the docs value (in this case, to `example`).

```suggestion
        example = lib.literalExpression ''
          {
            "treefmt" = null;
            "pyproject-fmt" = pkgs.python312Packages.pyproject-parser;
          }
        '';
```